spec/fetcher.spec.js

```diff
@@ -162,12 +160,15 @@ describe('Fetcher', () => {
       });
 
       it('should rejects with error', () => {
-        let result = fetcher.fetchVAST({
-          url: url,
-          maxWrapperDepth: 5,
-          emitter: () => {},
-        });
-        return expect(result).rejects.toEqual(new Error('timeout'));
+        let result = fetcher
+          .fetchVAST({
+            url: url,
+            maxWrapperDepth: 5,
+            emitter: () => {},
+          })
+          .catch(() => {
+            return expect(result).rejects.toEqual('error');
```

Comment:
Is the error type different here ? Because it should be iso with was we had before in term of behavior